platform/android/CHANGELOG.md

```diff
@@ -3,23 +3,31 @@
 
 ## master
-- Enable variable label placement when `text-allow-overlap` property is set to true [#15354](https://github.com/mapbox/mapbox-gl-native/pull/15354).
-- Introduce `text-writing-mode` layout property for symbol layer [#14932](https://github.com/mapbox/mapbox-gl-native/pull/14932). The `text-writing-mode` layout property allows control over symbol's preferred writing mode. The new property value is an array, whose values are enumeration values from a ( `horizontal` | `vertical` ) set.
+
+## 8.3.0-alpha.3 - August 15, 2019
+[Changes](https://github.com/mapbox/mapbox-gl-native/compare/android-v8.3.0-alpha.2...android-v8.3.0-alpha.3) since [Mapbox Maps SDK for Android v8.3.0-alpha.2](https://github.com/mapbox/mapbox-gl-native/releases/tag/android-v8.3.0-alpha.2):
+
+### Gestures handling changes
+ - Bumped gestures library version to v0.5.0 which replaces the compat gesture detector with a custom scale gesture detector implementation. This is a softly breaking change because the underlying scale gesture detector reference has been removed. This brings a set of improvements to gestures without removing any functionality from the previous implementation. [#15136](https://github.com/mapbox/mapbox-gl-native/pull/15136)
+ - Refined thresholds and velocity animations of scale and rotate gestures [#15136](https://github.com/mapbox/mapbox-gl-native/pull/15136)
+ - Execute quick-zoom scale change based on the Y axis delta change. This makes the quick-zoom changes linear. [#15386](https://github.com/mapbox/mapbox-gl-native/pull/15386)
 
 ### Features
- - Add fallback support to local ideograph font families [#15255](https://github.com/mapbox/mapbox-gl-native/pull/15255)
+ - Introduced `text-writing-mode` layout property for symbol layer. The text-writing-mode layout property allows for labels in a symbol layer to be placed horizontally or vertically. The new property value is an array, whose values are enumeration values from a ( `horizontal` | `vertical` ) set. [#14932](https://github.com/mapbox/mapbox-gl-native/pull/14932)
+ - Added fallback support to local ideograph font families [#15255](https://github.com/mapbox/mapbox-gl-native/pull/15255)
 
 ### Bug fixes
  - Fixed an issue where it was possible to set the map’s content insets then tilt the map enough to see the horizon, causing performance issues [#15195](https://github.com/mapbox/mapbox-gl-native/pull/15195)
  - Allow loading of a map without a Style URI or Style JSON [#15293](https://github.com/mapbox/mapbox-gl-native/pull/15293)
  - Fixed an issue where animated camera transitions zoomed in or out too dramatically [#15281](https://github.com/mapbox/mapbox-gl-native/pull/15281)
+ - Enable variable label placement when `text-allow-overlap` property is set to true [#15354](https://github.com/mapbox/mapbox-gl-native/pull/15354)
 
 ## 8.3.0-alpha.2 - August 7, 2019
```

Comment:
hm, are you also updating the alpha.2 changelog in this pr? 🤔

Reply:
Only indentation on those 2 entries.